libs/documentation/src/lib/pages/introduction/introduction.component.ts

```diff
@@ -1,18 +1,17 @@
 import { Component, OnInit, ViewEncapsulation } from '@angular/core';
```

Comment:
Could you also check the html file for this component for references to layouts?

Reply:
fixed and removed all references